### ISSUE TITLE
Sortable freezes when acceptFrom list does not exist

### ISSUE THREAD

**Issue by lukasoppermann** [lukasoppermann/html5sortable#520]
Originates from #519 

## Current behavior:
When `acceptFrom` is used, but no sortable with the selector can be found the sortable freezes.
**Needs validation:** This should be validated with an example on codepen.

**Desired behavior:**
1. Catch error from [isConnected](https://github.com/lukasoppermann/html5sortable/blob/master/src/isConnected.ts) and log to console.
2. Make sure the sortables work as if `acceptFrom` was not present

**Comment by kaffarell** [comment]
I would like to work on this but I was not able to reproduce this. What do you mean by freezes?
I tried to modify the [example](https://lukasoppermann.github.io/html5sortable/index.html) by supplying a non-existing selector to  acceptFrom but it simply didn't accept elements from the other list. The original list didn't changed at all and I was still able to sort the elements in it... Maybe I am doing something wrong...

**Comment by kaffarell** [comment]
I created a [codepen](https://codepen.io/kaffarell/pen/eYJGKxj) (The sortable call is at the bottom of the JS file)
```js
sortable('.ul', {
      forcePlaceholderSize: true,
      placeholderClass: 'ph-class',
      hoverClass: 'bg-maroon yellow'
});

sortable('.ul1', {
      acceptFrom: '.ul1, .ulas',
      forcePlaceholderSize: true,
      placeholderClass: 'ph-class',
      hoverClass: 'bg-maroon yellow'
})
```
This is the call to `sortable()`. `'.ulas'` is an invalid selector

**Comment by lukasoppermann** [comment]
Hey @kaffarell I currently can't reproduce it either. We should read through #519 again and see if there are any more insights on how to reproduce it. If we can't reproduce it, we can close this one.

I think if you change the code to this:

```
sortable('.ul', {
      acceptFrom: '.ulas',
      forcePlaceholderSize: true,
      placeholderClass: 'ph-class',
      hoverClass: 'bg-maroon yellow'
})
```

The issue arises. I think the issue is that if you don't include the lists own name, it does not allow you to drop and item into it.
It has nothing to do with the list not being there or not.

So this needs to be updated in the documentation and we must consider if this needs to be improved. @kaffarell can you document it in the readme?

**Comment by kaffarell** [comment]
👍 Going to change the documentation soon, can I close this issue?

**Comment by lukasoppermann** [comment]
Hey @kaffarell please leave it open until the documentation is updated. In the PR to update the readme you can all "fixes #520" and this issue should auto-close once it is merged. Just FYI [closing-issues-using-keywords](https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords#:~:text=issue%20will%20close.-,Closing%20an%20issue%20in%20the%20same%20repository,merged%20into%20the%20default%20branch) (I also don't know all keywords, but it is very handy, you can also close multiple issues this way)
